seqr/views/utils/permissions_utils.py

```diff
@@ -26,7 +43,9 @@ def has_project_permissions(project, user, can_edit=False, is_owner=False):
     if is_owner:
         permission_level = IS_OWNER
 
-    return user.has_perm(permission_level, project) or (user.is_staff and not project.disable_staff_access)
+    if anvil_has_perm(user, permission_level, project) or (user.is_staff and not project.disable_staff_access):
+        return True
+    return user.has_perm(permission_level, project)
```

Comment:
`return user.has_perm(permission_level, project) or (user.is_staff and not project.disable_staff_access) or anvil_has_perm(user, permission_level, project)`

Reply:
Got it. Thanks.